### PR TITLE
V1.0

### PR DESCRIPTION
**Pull request by xrotwang**
I ran all commands in the release instructions (twice, but the second run didn't change anything - as was expected).

So
- `app` removed
- some datasets re-mapped
- README.md adapted
- a couple of fixes

### CODE REVIEW

File: datasets/Babelnet/Babelnet.tsv-metadata.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
             "datatype": "integer"
           }, 
           {
-            "name": "BABELNET_GLOSS",
+            "name": "ENGLISH",
```
Comment (xrotwang) [inline datasets/Babelnet/Babelnet.tsv-metadata.json:26]: This makes sure the Babelnet glosses are picked up as well, as forms to which the definition and ID are related.

File: README.md [hunk 2]
```diff
@@ -87,69 +88,42 @@ If you can't find the `catalog.ini` file, create a directory `mkdir /Users/YOURN
 
 ## Mapping procedure
 
-### Create a map.py file
+"Mapping a dataset" refers to the process of mapping the words or concepts described in a dataset to Concepticon's
+concept sets and extracting associated norms, ratings or relations from a dataset's "raw" data.
 
-You need to derive a dataset first:
+This is facilitated small python modules (one per dataset), located at `datasets/<dataset-ID>/norare.py`. These
```
Comment (AnnikaTjuka) [inline README.md:94]: There is a "with" missing: "facilitated with" And I think we usually write "Python" in upper case.

File: README.md [hunk 3]
```diff
@@ -162,14 +136,16 @@ Don't forget to add the Conceticon columns:
 
 ### Download and map
 
-Make sure that you have stored the map.py and metadata.json files according to the schema provided for the other data sets and created a `raw` folder in your data set folder. If you are not already, navigate to the `norare-data` folder and type the following commands into your terminal:
+Make sure that you have stored the `norare.py` and metadata.json files according to the schema provided for the other
```
Comment (AnnikaTjuka) [inline README.md:139]: Add `` to metadata.json? "...and `metadata.json` files..."

File: RELEASING.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- validation
+  ```shell
+  norare --repos ../concepticon-data/ --norarepo . validate
+  ```
+- summary statistics
+  ```shell
+  norare --repos ../concepticon-data/ --norarepo . stats
+  ```
+- make sure `cldfbench makecldf` succeeds.
+- Create the release commit:
```
Comment (AnnikaTjuka) [inline RELEASING.md:21]: I would use lowercase letters at the beginning consistently in the list:
- create ...
- create...
- push ...
- create ...

File: RELEASING.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  ```shell
+  git commit -a -m "release <VERSION>"
+  ```
+- Create a release tag:
+  ```
```
Comment (AnnikaTjuka) [inline RELEASING.md:26]: I don't see a difference in the output but the other commands start with " ```shell ".

File: RELEASING.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  git tag -a v<VERSION> -m"<VERSION> release"
+  ```
+- Push to github:
```
Comment (AnnikaTjuka) [inline RELEASING.md:29]: "GitHub"

File: datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv [hunk 7]
```diff
@@ -152,8 +152,8 @@ CONCEPTICON_ID	CONCEPTICON_GLOSS	SPANISH	SPANISH_FREQUENCY	SPANISH_FREQUENCY_PM
 1231	NEW	nuevo	964.0	303.7177063642092	2.984527313343793
 1233	NIGHT	noche	1301.0	409.8928796471329	3.114610984232173
 1234	SMOOTH	liso	7.0	2.205419029615627	0.9030899869919436
-1235	WHO	quién	1535.0	483.6168872085696	3.186391215695493
-1236	WHAT	que	149996.0	47257.71896660366	5.176082573079183
+1235	WHO	quien	815.0	256.7737870195337	2.911690158753861
```
Comment (AnnikaTjuka) [inline datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv:155]: Interesting, the accents seem to not have been picked up in the second run.

File: datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv [hunk 8]
```diff
@@ -72,15 +72,15 @@ CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH	PAVIO_NORMS	ENGLISH_IMAGEABILITY_MEAN	E
...
-1215	THOU	YOU	0	3.31	6.96	2305
 1216	MOTHER	MOTHER	593	6.67	6.86	1363
 1218	ROPE	CORD	198	6.03	4.82	467
 122	CART	WAGON	0	0.00	4.83	2225
 1221	NOSE	NOSE	0	0.00	6.27	1418
 1224	LIVER	LIVER	0	5.38	4.86	1238
-1227	BOARD	BOARD	99	6.07	6.06	230
-1228	EARTH (SOIL)	EARTH	276	6.27	6.47	641
+1227	BOARD	PLANK	668	6.30	5.23	1564
+1228	EARTH (SOIL)	SOIL	791	6.00	5.75	1930
```
Comment (AnnikaTjuka) [inline datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv:83]: There seems to be a pattern in the list that the second choice was used or is this just a coincidence? The previous mappings seem more "correct' than the current ones. See also ANSWER changed to REPLY, PRISON changed to JAIL, BOWL changed to MUG.

**Review by AnnikaTjuka — COMMENTED** [review]
Many thanks for putting all this together! I left a few comments. The last comments about the mappings are just out of curiosity.

File: datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv (hunk 7 above)
Comment (xrotwang) [reply datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv:155]: `quien` appears first in the raw data, so when `quién` is encountered, we already have a match for `WHO`.

File: datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv (hunk 8 above)
Comment (xrotwang) [reply datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv:83]: Hm. I'd need to check whether we compute the available mappings somewhat differently now.

File: datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv (hunk 7 above)
Comment (AnnikaTjuka) [reply datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv:155]: I see.

File: datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv (hunk 8 above)
Comment (xrotwang) [reply datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv:83]: Good catch. It seems I don't sort as I should here
https://github.com/concepticon/pynorare/blob/58f0638aaf6a77a4ec6ca1116cdebd5aad2aa31c/src/pynorare/api.py#L212
missing a `-` before `_PRIORITY`.

Comment (xrotwang) [reply datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv:83]: `pynorare` needs a bugfix release anyway :)

Comment (xrotwang) [reply datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv]: Just pushed the data re-mapped with fixed priority. Looks better :)

File: datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv (hunk 7 above)
Comment (xrotwang) [reply datasets/Alonso-2011-OralFrequency/Alonso-2011-OralFrequency.tsv:155]: Position in the data only wins when the matches have the same priority, though. But that's the case here:
```
$ grep WHO raw/concepticon-data/mappings/map-es.tsv 
1235	WHO///quien	2
1235	WHO///quién	2
1235	WHO///quién?	2
1235	WHO///¿quién?	1
```

File: datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv (hunk 8 above)
Comment (AnnikaTjuka) [reply datasets/Clark-2004-ImageryFamiliarity/Clark-2004-ImageryFamiliarity.tsv]: Great!

**Review by AnnikaTjuka — APPROVED** [review]
Looks good from my side!

**Review by LinguList — APPROVED** [review]
Thanks a lot! It is interesting to see how the mapping slowly change with increased data. So far, I could only see that this is making them better, if it is possible to say so.

**Comment by xrotwang** [comment]
I think it's essential that Concepticon is "hand-curated". Otherwise - with two automated guesses stacked on top of each other - NoRarRe wouldn't be possible.
With the workflow as is, on the other hand, NoRaRe, and changes between versions of NoRaRe turns into a quality check for Concepticon :)

**Pull request merged**